src/data/nat/choose/multinomial.lean

```diff
@@ -0,0 +1,171 @@
+/-
+Copyright (c) 2022 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+Heavily inspired by code from Kyle Miller and Kevin Buzzard
+-/
+import algebra.big_operators.basic
+import algebra.big_operators.fin
+import data.nat.choose.basic
+import data.finset.basic
+import data.list.perm
+import data.fin.vec_notation
+
+import tactic.linarith
```

Comment:
```suggestion
import algebra.big_operators.basic
import algebra.big_operators.fin
import data.nat.choose.basic
import data.finset.basic
import data.list.perm
import data.fin.vec_notation
```
Many are probably unnecessary (may be necessary later, e.g. `data.list.perm`). `data.finset.basic` should already be imported by `algebra.big_operators.basic`.

Reply:
I think the imports could be reduced to

```
import algebra.big_operators.fin
import data.nat.choose.basic
import data.fin.vec_notation

import tactic.linarith
```

Without the import tactic.linarith I get errors on "linarith" and "ring", did you remove it on purpose? If yes, should I be doing something else to fix that?

Reply:
I think I removed the only use of `linarith`, but I didn't think about `ring`; maybe `import tactic.ring` works?